### PR TITLE
Replace install-gpu-drivers script with chwd autoconfiguration

### PR DESCRIPTION
**Pull request by ventureoo**
This makes debugging a bit easier for us, as it allows to reproduce errors related to driver setup after installation.

### CODE REVIEW

**Comment by vnepogodin** [comment]
idk about cases where we might want to install multiple profiles. in that case if intel igpu is upper in the list to be picked, then it will install only for intel igpu and nothing will be installed for nvidia dgpu (same for amd)

File: src/modules/chwd/main.py [hunk 1]
```diff
@@ -75,7 +75,7 @@ def run():
                 "exist, doing nothing".format(root_mount_point))
 
     # run the command in chroot
-    shell_command = ["arch-chroot", root_mount_point, "/etc/calamares/scripts/install-gpu-drivers"]
+    shell_command = ["arch-chroot", root_mount_point, "chwd --autoconfigure"]
```
Comment (vnepogodin) [inline src/modules/chwd/main.py]:
```suggestion
    shell_command = ["arch-chroot", root_mount_point, "chwd", "--autoconfigure"]
```

arch-chroot pass arguments as-is

**Review by vnepogodin — CHANGES_REQUESTED** [review]

**Review by vnepogodin — APPROVED** [review]
LGTM

was confirmed that its working

**Comment by ventureoo** [comment]
Has been manually merged (https://github.com/CachyOS/cachyos-calamares/commit/26c120952c9c97867d9c51862875dcd6248f3c96), so closing.

**Pull request closed without merging**